### PR TITLE
Rotating Wing State and Scheduling updates

### PR DESCRIPTION
**Pull request by tmldeponti**
In the past months I have been reworking how the effectiveness should be scheduled for the rotating wing drone. The idea is to have a more straightforward approach which can be more easily scaled to the bigger drone. 
At the same time, I have been working on cleaning up the RW state machine such that it is independent of stabilization and guidance files so that it can be used by other controllers alike the oneloop. 

In order to minimize conflict with the current rotating wing controller, these reworks are proposed as a "V2" of the already existing files. In the next months I will then merge the two versions keeping the best of both options. In the mean time though it would be useful to have them both and be able to pick and choose between test flights.

### CODE REVIEW

**Comment by gautierhattenberger** [comment]
The test on the `eff_scheduling_rot_wing_V2` module is failing, but beside that, I don't mind merging this, especially if you merge or choose one of your solutions in the end.

**Comment by tmldeponti** [comment]
> You can manually test this on your computer as well if you run `make test_modules`, to make sure everything works. This way you don't have wait on the CI, and use up all of the compute time

Thanks! that indeed would have saved me some time.

File: sw/airborne/modules/ctrl/eff_scheduling_rot_wing_V2.h [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+#define CTRL_EFF_SCHED_ROT_WING_H
+
+#include "std.h"
+#include "generated/airframe.h"
+
+#ifndef EFF_MAT_ROWS_NB
+#define EFF_MAT_ROWS_NB 6
+#endif
+
+#define aX 0
```
[the remark this answers is not in the document]
Comment (tmldeponti) [reply sw/airborne/modules/ctrl/eff_scheduling_rot_wing_V2.h:36]: They are used to uniqly define the structure of the effectiveness matrix. An axis gets assigned a value, such that one can use the axis name to define the effectiveness.

[the remark this answers is not in the document]
Comment (tmldeponti) [reply sw/airborne/modules/ctrl/eff_scheduling_rot_wing_V2.h]: I have made them more specific to my module and also added an explanation

**Review by EwoudSmeur — APPROVED** [review]
Nice work!

**Pull request merged**